pkgs/test/nixpkgs-check-by-name/workflows/check-by-name/fetch-latest-version-of-channel.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
+echo "Fetching latest version of channel $channel"
+# This is probably the easiest way to get Nix to output the path to a downloaded channel!
+nixpkgs=$(nix-instantiate --find-file nixpkgs -I nixpkgs=channel:"$channel")
+# This file only exists in channels
+toolingBinariesSha=$(<"$nixpkgs"/.git-revision)
+echo "Channel $channel is at revision $toolingBinariesSha"
+echo "nixpkgs=$nixpkgs" >> "$GITHUB_ENV"
+echo "toolingBinariesSha=$toolingbinariesSha" >> "$GITHUB_ENV"
```

Comment:
You can remove all `GITHUB_ENV` assignments if we only have a single step and script.